lib/collection/proxy-config.js

```diff
@@ -1,8 +1,13 @@
 var _ = require('../util').lodash,
     Property = require('./property').Property,
-    Url = require('./url').Url,
     UrlMatchPattern = require('../url-pattern/url-match-pattern').UrlMatchPattern,
-
+    DEFAULT_PROTOCOLS = ['http'],
+    DEFAULT_PORT = 8080,
```

Comment:
Why do this for integers?

The V8 Engine is smart enough to inline the integers to machine code when it sees them. By making variables, we're adding an extra lookup.

It's not a big deal, but if we don't need it, might as well not do it.